src/php/docker/alpine/Dockerfile

```diff
@@ -21,15 +21,19 @@ ARG MAKEFLAGS=-j8
 
 WORKDIR /tmp
 
-RUN wget https://phar.phpunit.de/phpunit-9.5.9.phar &&   mv phpunit-9.5.9.phar /usr/local/bin/phpunit &&   chmod +x /usr/local/bin/phpunit
+RUN wget https://phar.phpunit.de/phpunit-9.5.9.phar && \
```

Comment:
If there are problems with the inja changes, it'd be a shame to roll back unrelated formatting changes. Can you do those separately?

Reply:
Looks like I got to this review too late. 🤞